src/v2/Utils/logger.ts

```diff
@@ -8,6 +8,7 @@ export default function createLogger(namespace = "reaction") {
 
   return {
     log: (...messages) => {
```

Comment:
Actually, you know... instead of doing `console.info`, `console.warn`, or `console.error` we should _probably_ be using this logger. Though, funnily enough we should update the namespace to be `"force"` instead of `"reaction"` as it is now.